pkg/configuration/configuration.go

```diff
@@ -260,9 +260,9 @@ func (c *Config) GetEnvironment() string {
 	return c.host.GetString(varEnvironment)
 }
 
-// GetMasterUserRecordUpdateRetries returns the number of retries before failing to update a MasterUserRecord
-func (c *Config) GetMasterUserRecordUpdateRetries() int {
-	return c.host.GetInt(varMasterUserRecordUpdateRetries)
+// GetMasterUserRecordUpdateFailureThreshold returns the failure threshold for MUR updates. When a MUR update fails, we will re-try N times before giving up and stopping the entire tier update process.
```

Comment:
This (my) comment is not good anymore. Sorry if I confused you but my suggestion was either clarify the language (so, I suggested this comment) or change the logic (what you did), so it's not the overall number of attempts and not the number of re-tries after the first failure (if any). So, your new comment "specifies the number allowed failures before stopping trying to update a MasterUserRecord" is what we need here too :)

Again, sorry for the confusion :(

```suggestion
// GetMasterUserRecordUpdateFailureThreshold returns the number of allowed failures before stopping trying to update a MasterUserRecord
```

Reply:
no worries, it's also my fault. I made the changes in the evening and I was a bit in a hurry...